baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ObjectsEnforceLogSafeUse.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.errorprone;
+
+import com.google.auto.service.AutoService;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.matchers.Matcher;
+import com.google.errorprone.matchers.Matchers;
+import com.google.errorprone.matchers.method.MethodMatchers;
+import com.sun.source.tree.ExpressionTree;
+import com.sun.source.tree.MethodInvocationTree;
+import java.util.regex.Pattern;
+
+@AutoService(BugChecker.class)
+@BugPattern(
+        name = "ObjectsEnforceLogSafeUse",
+        category = BugPattern.Category.ONE_OFF,
+        severity = BugPattern.SeverityLevel.ERROR,
+        summary = "Objects.requireNonNull(Object) and Objects.requireNonNull(Object, String) methods with a "
+                + "constant message and no parameters should use logsafe Preconditions.checkNotNull(Object, String)")
```

Comment:
might as well give the FQN: `com.palantir.logsafe.Preconditions#checkNotNull(Object, String)`